lab-brianna/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "lab-brianna",
+  "version": "1.0.0",
+  "description": "whomp",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
**TODO FOR LOST POINTS**

 You need a test script in your package.json per the instructions.